Source/ImagePickerController.swift

```diff
@@ -217,6 +217,11 @@ public class ImagePickerController: UIViewController {
       selector: #selector(volumeChanged(_:)),
       name: "AVSystemController_SystemVolumeDidChangeNotification",
       object: nil)
+
+    NSNotificationCenter.defaultCenter().addObserver(self,
```

Comment:
Hi, it is already handled in `deinit` https://github.com/hyperoslo/ImagePicker/blob/master/Source/ImagePickerController.swift#L195